mobius/src/algorithms/orbit_tile.rs

```diff
@@ -0,0 +1,182 @@
+use abstraction::{Group, GroupAction};
+
+use crate::{isogonal::Isogonal, Complex};
+
+#[derive(Clone)]
+pub struct OrbitTile<G, P> {
```

Comment:
This would be a good candidate for an explainer diagram. Make one before I merge this.